poker/src/components/ConnectToLobby/ConnectToLobby.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC, useState, useEffect } from 'react';
+import Popup from 'reactjs-popup';
+import 'reactjs-popup/dist/index.css';
+import c from './ConnectToLobby.module.scss';
+
+interface errorProps {
+    firstName: string;
+    lastName: string;
+    jobPosition: string;
+}
+
+const ConnectToLobby: FC = () => {
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [jobPosition, setJobPosition] = useState<string>('');
+  const [error, setError] = useState<errorProps>({ firstName: '', lastName: '', jobPosition: '' });
+  const [observer, setObserver] = useState<boolean>(false);
+  const [image, setImage] = useState<string>('');
+
+  useEffect(() => {
+    validate();
+  }, [firstName, lastName, jobPosition]);
+
+  const validate = () => {
+    if (firstName === '') {
+      setError((state) => ({ ...state, firstName }));
+    }
+    if (lastName === '') {
+      setError((state) => ({ ...state, lastName }));
+    }
+    if (jobPosition === '') {
+      setError((state) => ({ ...state, jobPosition }));
+    }
+  };
+
+  const handelSubmit = () => {
+    console.log('send form');
```

Comment:
add event.preventDefault();

Reply:
lastName and jobPosition optional fields and they must not have validation